pan_firewall/assets/logs/pan.firewall.yaml

```diff
@@ -230,4 +385,1028 @@ pipeline:
+                query: "@severity:high"
+              name: "4"
+            - filter:
+                query: "@severity:critical"
+              name: "5"
+          target: ocsf.severity_id
+        - type: category-processor
+          name: Severity Name Mapping
+          enabled: true
+          categories:
+            - filter:
+                query: "@severity:informational"
+              name: Informational
+            - filter:
+                query: "@severity:low"
+              name: Low
+            - filter:
+                query: "@severity:medium"
+              name: Medium
+            - filter:
+                query: "@severity:high"
+              name: High
+            - filter:
+                query: "@severity:critical"
+              name: Critical
+          target: ocsf.severity
+        - type: string-builder-processor
+          name: 1 - in attribute ocsf.activity_id
+          enabled: true
+          template: "1"
+          target: ocsf.activity_id
+          replaceMissing: false
+        - type: string-builder-processor
+          name: Create - in attribute ocsf.activity_name
+          enabled: true
+          template: Create
+          target: ocsf.activity_name
+          replaceMissing: false
+        - type: string-builder-processor
+          name: 2 - in attribute ocsf.category_uid
+          enabled: true
+          template: "2"
+          target: ocsf.category_uid
+          replaceMissing: false
+        - type: string-builder-processor
+          name: Findings - in attribute ocsf.category_name
+          enabled: true
+          template: Findings
+          target: ocsf.category_name
+          replaceMissing: false
+        - type: string-builder-processor
+          name: 2 - in attribute ocsf.status_id
+          enabled: true
+          template: "2"
+          target: ocsf.status_id
+          replaceMissing: false
+        - type: string-builder-processor
+          name: 2004 - in attribute ocsf.class_uid
+          enabled: true
+          template: "2004"
+          target: ocsf.class_uid
+          replaceMissing: false
+        - type: string-builder-processor
+          name: Detection Finding - in attribute ocsf.class_name
+          enabled: true
+          template: Detection Finding
+          target: ocsf.class_name
+          replaceMissing: false
+        - type: string-builder-processor
+          name: Firewall - in attribute ocsf.finding_info.product.name
+          enabled: true
+          template: Firewall
+          target: ocsf.finding_info.product.name
+          replaceMissing: false
+        - type: string-builder-processor
+          name: Palo Alto - in attribute ocsf.finding_info.product.vendor_name
+          enabled: true
+          template: Palo Alto
+          target: ocsf.finding_info.product.vendor_name
+          replaceMissing: false
+        - type: string-builder-processor
+          name: 0 - in attribute ocsf.confidence_id
```

Comment:
If we are doing `ocsf.confidence_id`, we should do the sibling attribute `ocsf.confidence`